api-reference/v1.0/resources/domain.md

```diff
@@ -0,0 +1,92 @@
+# domain resource type
+
+Represents a domain associated with the tenant.
+
+Use domain operations to associate domains to a tenant, verify domain ownership, and configure supported services.  Domain operations enable registrars to automate domain association for services such as Office 365. For example, as part of domain sign up, a registrar can enable a vanity domain for email, websites, authentication, etc.
+
+To associate a domain with a tenant:
+
+1. [Associate](../api/domain_post_domains.md) a domain with a tenant.
+
+2. [Verify](../api/domain_list_verificationdnsrecords.md) ownership of the domain. Verification records can be added to the domain through the domain registrar or DNS server configuration.
```

Comment:
You are missing a step here I think.  First you need to call verificationdnsrecords.  This give you the info you need to add to your domain entry in your domain registrar.  **Then** once update, you need to call the verify method on the domain (domain/{id}/verify) for us to call out and verify that you actually own the DNS record/domain.  Once verified we flip the domain state to verified.  
This needs to be fixed in /beta doc too.